[2022-07-21 01:06:36,379] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn scheduled__2022-07-20T00:00:00+00:00 [queued]>
[2022-07-21 01:06:36,387] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn scheduled__2022-07-20T00:00:00+00:00 [queued]>
[2022-07-21 01:06:36,387] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-21 01:06:36,387] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-21 01:06:36,388] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-21 01:06:36,406] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): setup_http_conn> on 2022-07-20 00:00:00+00:00
[2022-07-21 01:06:36,412] {standard_task_runner.py:52} INFO - Started process 608 to run task
[2022-07-21 01:06:36,414] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'process_files', 'setup_http_conn', 'scheduled__2022-07-20T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/process_files.py', '--cfg-path', '/tmp/tmp4_kulgxo', '--error-file', '/tmp/tmp7m6y9imj']
[2022-07-21 01:06:36,415] {standard_task_runner.py:80} INFO - Job 3: Subtask setup_http_conn
[2022-07-21 01:06:36,478] {task_command.py:371} INFO - Running <TaskInstance: process_files.setup_http_conn scheduled__2022-07-20T00:00:00+00:00 [running]> on host 4534be881ccc
[2022-07-21 01:06:36,559] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process_files
AIRFLOW_CTX_TASK_ID=setup_http_conn
AIRFLOW_CTX_EXECUTION_DATE=2022-07-20T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-20T00:00:00+00:00
[2022-07-21 01:06:36,560] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-21 01:06:36,560] {subprocess.py:74} INFO - Running command: ['bash', '-c', "*** connections delete 'http_api'\n*** connections delete 'pg_conn'\n\n\n*** connections add 'http_api' --conn-type 'http' --conn-host 'raw.githubusercontent.com/tadinve/EKG-Foundations/master/04A-postgress/'\n\n\n*** connections add pg_conn \\\n  --conn-type 'postgres' \\\n  --conn-host 'my-postgres' \\\n  --conn-login 'postgres' \\\n  --conn-schema 'northwind' \\\n  --conn-password 'postgres' \n\nexit 0"]
[2022-07-21 01:06:36,572] {subprocess.py:85} INFO - Output:
[2022-07-21 01:06:39,149] {subprocess.py:92} INFO - Did not find a connection with `conn_id`=http_api
[2022-07-21 01:06:41,989] {subprocess.py:92} INFO - Did not find a connection with `conn_id`=pg_conn
[2022-07-21 01:06:46,894] {subprocess.py:92} INFO - [[34m2022-07-21 01:06:46,893[0m] {[34mproviders_manager.py:[0m218} INFO[0m - Optional provider feature disabled when importing '***.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-***-providers-google' package[0m
[2022-07-21 01:06:47,838] {subprocess.py:92} INFO - Successfully added `conn_id`=http_api : http://:@raw.githubusercontent.com/tadinve/EKG-Foundations/master/04A-postgress/:
[2022-07-21 01:06:51,619] {subprocess.py:92} INFO - [[34m2022-07-21 01:06:51,618[0m] {[34mproviders_manager.py:[0m218} INFO[0m - Optional provider feature disabled when importing '***.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-***-providers-google' package[0m
[2022-07-21 01:06:51,874] {subprocess.py:92} INFO - [[34m2022-07-21 01:06:51,874[0m] {[34mcrypto.py:[0m82} WARNING[0m - empty cryptography key - values will not be stored encrypted.[0m
[2022-07-21 01:06:51,888] {subprocess.py:92} INFO - Successfully added `conn_id`=pg_conn : postgres://postgres:******@my-postgres:/northwind
[2022-07-21 01:06:52,501] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-21 01:06:52,543] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=process_files, task_id=setup_http_conn, execution_date=20220720T000000, start_date=20220721T010636, end_date=20220721T010652
[2022-07-21 01:06:52,582] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-21 01:06:52,621] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
